test/fixtures/password_resets.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
Please document these fixtures for which tests they are used for.

Reply:
You may need some internal documentation page aside from inlined documentation (e.g., some section of `README.md` or some other `.md` similar for such medium scope documentation (e.g., where are the constants defined, what is `cover` and what fixtures are defined, what tests are defined.